AppDashboard/lib/datastore_viewer.py

```diff
@@ -0,0 +1,637 @@
+""" Implements the datastore viewer interface. """
+import math
+import urllib
+
+from app_dashboard import AppDashboard
+from datastore_location import DATASTORE_LOCATION
+from google.appengine.api import datastore
+from google.appengine.api import memcache
+from google.appengine.api.datastore_distributed import DatastoreDistributed
+from google.appengine.api import api_base_pb
+from google.appengine.datastore import datastore_pb
+
+from google.appengine.tools.devappserver2.admin.datastore_viewer import (
+  _property_name_to_value,
+  DataType)
```

Comment:
should this parentheses be on the next line?

not hugely important, mostly curious

Reply:
Not sure. I didn't see anything in PEP 8 to indicate one way or the other.